.travis.yml

```diff
@@ -29,6 +29,8 @@ node_js:
 
 # http://docs.travis-ci.com/user/gui-and-headless-browsers/
 before_install:
+  - curl https://yarnpkg.com/install.sh -sSfL | bash -s --
```

Comment:
Isn't this going to slow down every build on travis since it will always install this?

Reply:
Yes, Copied the behavior from `wire-web-packages`. Just saw that the new version is used by @dependabot without those lines in the travis file as well. Let's try without.